awx_collection/plugins/doc_fragments/auth_plugin.py

```diff
@@ -18,42 +18,42 @@ class ModuleDocFragment(object):
     env:
     - name: CONTROLLER_HOST
     - name: TOWER_HOST
-    deprecated:
-    - removed_from_collection: 'awx.awx.controller'
-    - removed_in: '4.0.0'
-    - why: Collection name change
-    - alternative: 'CONTROLLER_HOST'
+      deprecated:
+        removed_from_collection: 'awx.awx'
+        removed_in: '4.0.0'
```

Comment:
While antsibull-docs accepts both `removed_in` and `version` (and internally normalizes to `removed_in`), ansible-core wants this to be called `version`:
```suggestion
        version: '4.0.0'
```